backend/app/api/sms.py

```
@@ -0,0 +1,240 @@
+from fastapi import APIRouter, Depends, HTTPException, status, Query
+from sqlalchemy.orm import Session
+from typing import Optional
+import math
+
+from app.core.database import get_db
+from app.services.sms_service import sms_service
+from app.crud.reservation import reservation_crud
+from app.schemas.sms import (
+    SMSSend, SMSBulkSend, SMS, SMSList, SMSFilter, 
```

Ruff (0.11.9)

`app.schemas.sms.SMSFilter` imported but unused

Remove unused import: `app.schemas.sms.SMSFilter`

(F401)